src/lib/rule2F.ts

```diff
@@ -138,6 +138,93 @@ function allowsNameFromContent(elem: HTMLElement): boolean {
 
 export const TEST_ONLY = {allowsNameFromContent};
 
+// See https://lists.w3.org/Archives/Public/public-aria/2017Jun/0057.html
+// for discussion of roles & tags that forbid name from content.
```

Comment:
Please explicitly note that this isn't included in 1.1 but as per that thread implementations should check this anyway